types/node/v16/dns.d.ts

```diff
@@ -313,7 +313,9 @@ declare module 'dns' {
     /**
      * Uses the DNS protocol to resolve `CAA` records for the `hostname`. The`addresses` argument passed to the `callback` function
      * will contain an array of certification authority authorization records
-     * available for the `hostname` (e.g. `[{critical: 0, iodef: 'mailto:pki@example.com'}, {critical: 128, issue: 'pki.example.com'}]`).
+     * available for the `hostname` (e.g. `[{
+     
```

Comment:
Smth wrong here.

Also need to fix in `.../ts4.8/dns.d.ts`, `.../v16/ts4.8/dns.d.ts` please.

Reply:
tangled up with the keyboard. fixed